pom.xml

```diff
@@ -65,6 +65,21 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
+      <dependency>
```

Comment:
These seem to be sneaking back in somehow?

Reply:
Removed in a commit @amorton -- this SHOULD be safe, as far as I know. Not planning doing any other changes but just wanted to resolve this as it keeps on popping up :)